src/test/run-rbd-unit-tests.sh

```diff
@@ -1,17 +1,25 @@
 #!/usr/bin/env bash
-set -ex
+set -xe
```

Comment:
Nit: why this change?

Reply:
@dillaman
Both not intentional, will revert 'm